pkg/runcfanotify/runcfanotify.go

```diff
@@ -175,14 +162,26 @@ func NewRuncNotifier(callback RuncNotifyFunc) (*RuncNotifier, error) {
 	}
 	n.runcBinaryNotify = runcBinaryNotify
 
+	runcFound := false
+
 	for _, runcPath := range runcPaths {
 		file := filepath.Join(hostRoot, runcPath)
-		err = runcBinaryNotify.Mark(unix.FAN_MARK_ADD, unix.FAN_OPEN_EXEC_PERM, unix.AT_FDCWD, file)
-		if err == nil {
-			log.Debugf("Checking %q: done", file)
```

Comment:
I think it is useful to print which path it is trying

Reply:
I changed the logic a bit to always print the paths.